apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -254,3 +271,39 @@ export const fetchWorkflowEventDecision = async ({
 
   return handleZodError(error, workflow);
 };
+
+export const createWorkflowRequest = async ({
+  workflowDefinitionId,
+  context,
+}: {
+  workflowDefinitionId: string;
+  context: TWorkflowById['context'];
+}) => {
+  const [workflow, error] = await apiClient({
+    method: Method.POST,
+    url: `${getOriginUrl(env.VITE_API_URL)}/api/v1/external/workflows/run`,
+    body: {
+      workflowId: workflowDefinitionId,
+      context,
+    },
+    schema: z.any(),
+  });
+
+  return handleZodError(error, workflow);
+};
+
+export const fetchWorkflowDefinition = async ({
```

Comment:
Let's ensure consistency with other fetchers is kept. 🙏🏼 
```suggestion
export const fetchWorkflowDefinitionById = async ({
```